strongbox-web-core/src/test/resources/etc/conf/strongbox.xml

```diff
@@ -296,7 +296,9 @@
                             data-store="file-system"
                             layout="npm"
                             type="proxy">
-                    <remote-repository url="https://registry.npmjs.org/"/>
```

Comment:
Shouldn't we drop this file too ?

Reply:
Yes, I believe we should be able to do so.

Reply:
Done.